src/go/rpk/pkg/cli/cmd/topic_test.go

```diff
@@ -66,13 +66,13 @@ func TestTopicCmd(t *testing.T) {
 		{
 			name:           "create should allow passing arbitrary topic config",
 			cmd:            topic.NewCreateCommand,
-			args:           []string{"San Francisco", "-c", "custom.config:value", "--config", "another.config:anothervalue"},
+			args:           []string{"San Francisco", "--topic-config", "custom.config:value", "--topic-config", "another.config:anothervalue"},
 			expectedOutput: "Created topic 'San Francisco'.\\nYou may check its config with\\n\\nrpk topic describe 'San Francisco'\\n\"\n",
 		},
 		{
 			name:           "create should allow passing comma-separated config values",
 			cmd:            topic.NewCreateCommand,
-			args:           []string{"San Francisco", "-c", "custom.config:value", "--config", "cleanup.policy:cleanup,compact"},
```

Comment:
is this change extra then?

Reply:
How so?

Reply:
You're testing the alias?

Reply:
Yep, and I also removed the wrong —config flag

Reply:
why are they both `-c` and not --topic-config

Reply:
That’s the flag’s alias. It was there before this change.

Reply:
ack